asdf/asdftypes.py

```diff
@@ -9,6 +9,9 @@
 from .types import (AsdfType, CustomType, format_tag, ExtensionTypeMeta,
                     _all_asdftypes)
 
+__all__ = ["join_tag_version", "split_tag_version", "AsdfType", "CustomType",
+    "format_tag", "ExtensionTypeMeta", "_all_asdftypes"]
```

Comment:
Should we leave `_all_asdftypes` out of this list?  With the leading underscore it doesn't seem like it's already part of the public API.